Pseudocode/Dynamic Programming/Bursting Balloons/Source Code.tex

```diff
@@ -0,0 +1,71 @@
+% Set the Page Layout
+\documentclass[12pt]{article}
+\usepackage[inner = 2.0cm, outer = 2.0cm, top = 2.0cm, bottom = 2.0cm]{geometry}
+
+
+% Package to write pseudo-codes
+\usepackage{algorithm}
+
+% Remove the 'end' at the end of the algorithm
+\usepackage[noend]{algpseudocode}
+
```

Comment:
In your entire code, the formatting is messed up. Remember to use the convention, one tab for each block (with proper nesting within them).  See the Gist for more examples.